batching/build.gradle

```diff
@@ -38,7 +38,7 @@ android {
         minSdkVersion 14
         targetSdkVersion 28
         versionCode 1
-        versionName "1.3.9"
+        versionName "1.3.10"
```

Comment:
Can you also bump up the versions for other module ? to keep versions in sync across all modules

Reply:
done